boa/src/builtins/map/mod.rs

```diff
@@ -29,6 +32,7 @@ impl BuiltIn for Map {
             .name(Self::NAME)
             .length(Self::LENGTH)
             .method(Self::set, "set", 2)
+            .method(Self::entries, "entries", 0)
```

Comment:
We can also implement the `keys` and `values` methods

Reply:
In a separate PR with their own tests.